libminifi/include/utils/ByteArrayCallback.h

```diff
@@ -51,22 +51,24 @@ class ByteInputCallBack : public InputStreamCallback {
       stream->read(reinterpret_cast<uint8_t*>(vec.data()), gsl::narrow<int>(stream->size()));
     }
 
-    ptr = reinterpret_cast<char*>(&vec[0]);
+    ptr = vec.data();
```

Comment:
Do we even need this `ptr`?  It seems to be write-only.

Reply:
I didn't check it, but you're right, it's unused. Removing it.